libs/react-helpers/src/lib/grid/vol-cell.tsx

```diff
@@ -15,23 +21,28 @@ export interface IVolCellProps extends ICellRendererParams {
 export const BID_COLOR = 'darkgreen';
 export const ASK_COLOR = 'maroon';
 
-export const Vol = React.memo(({ value, relativeValue, type }: VolProps) => {
-  if ((!value && value !== 0) || isNaN(Number(value))) {
-    return <div data-testid="vol">-</div>;
+export const Vol = React.memo(
+  ({ value, relativeValue, type, testId }: VolProps) => {
+    if ((!value && value !== 0) || isNaN(Number(value))) {
+      return <div data-testid="vol">-</div>;
+    }
+    return (
+      <div className="relative" data-testid={testId || 'vol'}>
+        <div
+          className={classNames('h-full absolute top-0', {
+            'left-0': type === VolumeType.bid,
+            'right-0': type === VolumeType.ask,
+          })}
+          style={{
+            width: relativeValue ? `${relativeValue}%` : '0%',
```

Comment:
Question (non-blocking): why use a style prop instead of classes here?

Reply:
color styles will be changed to classes in another PR